spec/controllers/votes_controller_spec.rb

```diff
@@ -1,4 +1,75 @@
 require 'rails_helper'
 
 RSpec.describe VotesController, type: :controller do
+    let(:user1) { create(:user_with_voteable_status) }
+    let(:user2) { create(:user_with_voteable_status) }
+    let(:user3) { create(:user_without_voteable_status) }
+    before do
+      create_list(:question, 5, user: user1)
+      create_list(:answer, 5, user: user2)
+    end
+
+  context "when user upvotes another user's resource" do
+    before do
+      request.headers['Authorization'] = "Token token=#{user2.refresh_token}"
+    end
+
+    it "upvotes another user's resource" do
+      post :upvote, {resource_name: 'questions', resource_id: 1}
+      expect(response.body).to eq "{\"response\":1}"
+    end
+
+    it "doesnt upvote user's own resource" do
+      post :upvote, {resource_name: 'answers', resource_id: 1}
+      expect(response.body).to include "You can't vote for your post"
+    end
+  end
+
+  context "when user downvotes another user's resource" do
+    before do
+      request.headers['Authorization'] = "Token token=#{user1.refresh_token}"
+    end
+
+    it "downvotes another user's resource" do
+      post :downvote, {resource_name: 'questions', resource_id: 1}
+      expect(response.body).to include "You can't vote for your post"
```

Comment:
The context is saying: "when a user downvotes **another user's** resource", but the expected response is saying **your post**